kmk/kmk_keyboard.py

```diff
@@ -279,20 +277,16 @@ def go(self, hid_type=HIDModes.USB, **kwargs):
                 del self.leader_dictionary[k]
 
         while True:
-            state_changed = False
```

Comment:
I actually make use of this feature in Bluetooth splits when it comes to power save. Can we keep thees? There are several instances below as well.